### PR TITLE
Minor tweaks in title and section heading

### PR DESCRIPTION
**Pull request by everthemore**

### CODE REVIEW

File: attendeeguide.md [hunk 1]
```diff
@@ -17,13 +18,14 @@ Guidelines
...
+1. Open the participants pane (we recommend doing this at the beginning of the talk).
+2. Click the “raise your hand” button. Your name, now with a raised hand symbol, will go to the top of the participants list, which the moderator will be monitoring.
+3. The moderator will alert the speaker, announce your name, and unmute your microphone.
+4. Ask your question!
+5. When the interaction ends, the moderator will mute your microphone and “lower your hand”.
 
 ### When to ask
 
 Short talks: after the talk ends.
+
```
Comment (akhmerov) [inline attendeeguide.md]:
```suggestion
Short talks: after the talk ends.  
```

double space is a markdown newline

**Comment by akhmerov** [comment]
Other than the newline looks good to merge.

File: attendeeguide.md (hunk 1 above)
Comment (everthemore) [reply attendeeguide.md]: Thought it looked nicer with a newline rather than a oneliner

**Comment by akhmerov** [comment]
Newline is correct, but having a blank line is a new paragraph. A line break is double trailing space.

**Pull request merged**